addons/website/static/tests/tours/snippet_table_of_content.js

```diff
@@ -28,28 +29,11 @@ registerWebsitePreviewTour('snippet_table_of_content', {
     edition: true,
 }, () => [
     ...insertSnippet({id: "s_table_of_content", name: "Table of Content", groupName: "Text"}),
-    {
-        content: "Drag the Text snippet group and drop it.",
-        trigger: '#oe_snippets .oe_snippet[name="Text"] .oe_snippet_thumbnail:not(.o_we_ongoing_insertion)',
-        run: "drag_and_drop :iframe #wrap",
-    },
-    {
-        content: "Click on the s_table_of_content snippet.",
-        trigger: ':iframe .o_snippet_preview_wrap[data-snippet-id="s_table_of_content"]',
-        run: "click",
-    },
-    // To make sure that the public widgets of the two previous ones started.
+    ...insertSnippet({id: "s_text_block", name: "Text", groupName: "Text"}),
+    ...clickOnSnippet({id: "s_table_of_content", name: "Table of Content"}),
+    goBackToBlocks(),
     ...insertSnippet({id: "s_banner", name: "Banner", groupName: "Intro"}),
-    {
-        content: "Drag the Intro snippet group and drop it.",
-        trigger: '#oe_snippets .oe_snippet[name="Intro"] .oe_snippet_thumbnail:not(.o_we_ongoing_insertion)',
-        run: "drag_and_drop :iframe #wrap",
-    },
```

Comment:
Why this step is not converted?

Reply:
In ` ...insertSnippet` its already done so i replaced this code with util method.

Reply:
Oh now i get it, i think i missed this step thinking the above step already does that, will update it.